lib/counterfeit/version.rb

```diff
@@ -1,3 +1,3 @@
 module Counterfeit
-  VERSION = '0.1.10'
+  VERSION = '0.1.30'
```

Comment:
why not `0.1.11` ?

Reply:
because on the Gemfile.lock it was `counterfeit (0.1.2)` despite being `0.1.10` here.
As a safety measure I bumped it to `0.1.30`

Reply:
@shideneyu j'ai bien 0.1.10 dans le Gemfile.lock

Reply:
no no, it was `0.1.2`

![image](https://user-images.githubusercontent.com/876824/88521883-e12d4900-cff5-11ea-9c87-fbd94a5440fc.png)


ok updated to `0.1.20`, hopefully it won't pose a problem

Reply:
`0.1.12` because `0.1.11` 's tag is already taken: https://github.com/ArizenHQ/counterfeit/tags